### ISSUE TITLE
Visual Studio does not hit breakpoints when debugging tests

### ISSUE THREAD

**Issue by notisrac** [Microsoft/nodejstools#1893]
When trying to run unit tests with Mocha set as the test framework, everything is fine. However, when I try to debug the tests, VS does not hit any of my breakpoints in the tests. It shows the white centered breakpoint markers with the exclamation marks, indicating, that the symbols could not be loaded. Furthermore the test I was trying to debug will fail, regardless of its actual result!

Could you please advise on this? Is this a config issue on my side?

##### Expected Behavior 
When running a test in debug mode, VS should hit the first breakpoint

##### Actual Behavior 
When running a test in debug mode, no breakpoints are hit, and the test will fail, regardless of its actual result

- NTVS Version: 1.4.11027.3
- Visual Studio Version: 15.6.1
- Node.js Version: v9.5.0
- Mocha Version: 5.0.4

Also experienced the same in VS 2015 (14.0.25431.01 Update 3) with NTVS 1.2.40726.00

##### Steps to Reproduce
I followed this guide: https://github.com/Microsoft/nodejstools/wiki/Test-Explorer
1.  Create a new "Blank Node.js Console Application" project (any other type should do as well)
2.  Install the Mocha npm package (locally)
3.  Add a new "JavaScript Mocha UnitTest file" - the default contents will do (I tried putting the file in the root of the project, and in a "test" folder, but did not made any difference)
4.  Make sure, the "TestFramework" property of the file is set to "Mocha"
5.  Set a breakpoint on the first line (but any other will do) 
6.  Build the app
7.  Open the test explorer, and run the first test ("Test 1") - it should pass
8.  Debug the same test - the breakpoint will not be hit, and the test will fail

**Comment by paulvanbrenk** [comment]
Debugging node 8+ is not supported on Visual Studio 2017 Update 6. This should be fixed in the near future, but I unfortunately don't have much influence on the release schedule. 

Debugging should work with Node 6.

**Comment by notisrac** [comment]
Do you mean only for tests? Because I can debug my code without any problem with the setup mentioned above. The only debugging issues I'm having is with the tests.

**Comment by paulvanbrenk** [comment]
yes, only for tests... they have a different code path, because of the way the tests are started, vs. the way an app is started.
